### DISCUSSION TITLE
Is there a way to feed a string into CLIP text encode?

### DISCUSSION THREAD

**Discussion by ewlun** [WASasquatch/was-node-suite-comfyui#170] [Q&A]
I've tried using text to conditioning, but it doesn't seem to work. At least not by replacing CLIP text encode with one. This is what I have right now, and it doesn't work https://ibb.co/wyVKg6n

**Comment by wutipong** [comment]
Text to conditioning seems to work to me. Here's my workflow.

![image](https://github.com/WASasquatch/was-node-suite-comfyui/assets/3213191/7c3368a5-cf6f-4548-abbd-c0912d1fb67d)

**Comment by WASasquatch** [comment]
You can also convert any CLIPTextEncode textbox into a input now.

**Reply by wutipong** [reply]
Just understand what you meant here. There's an option to convert Text to input at the CLIPTextEncode. 

![image](https://github.com/WASasquatch/was-node-suite-comfyui/assets/3213191/d32768af-6d80-40bd-8c21-cfaf04b5246d)

![image](https://github.com/WASasquatch/was-node-suite-comfyui/assets/3213191/2cd1db07-1941-4e2e-8205-3ab220e8e9c1)

**Comment by wutipong** [comment]
I've just tried some SDXL  and got an issue when using the "Text to Conditioning". Here's the screenshot.

![image](https://github.com/WASasquatch/was-node-suite-comfyui/assets/3213191/4878fe86-72c3-44a4-8910-bde6adcb2707)

And here's the log in the console.
```
!!! Exception during processing !!!
Traceback (most recent call last):
  File "H:\ComfyUI_windows_portable\ComfyUI\execution.py", line 151, in recursive_execute
    output_data, output_ui = get_output_data(obj, input_data_all)
  File "H:\ComfyUI_windows_portable\ComfyUI\execution.py", line 81, in get_output_data
    return_values = map_node_over_list(obj, input_data_all, obj.FUNCTION, allow_interrupt=True)
  File "H:\ComfyUI_windows_portable\ComfyUI\execution.py", line 74, in map_node_over_list
    results.append(getattr(obj, func)(**slice_dict(input_data_all, i)))
  File "H:\ComfyUI_windows_portable\ComfyUI\nodes.py", line 1206, in sample
    return common_ksampler(model, seed, steps, cfg, sampler_name, scheduler, positive, negative, latent_image, denoise=denoise)
  File "H:\ComfyUI_windows_portable\ComfyUI\nodes.py", line 1176, in common_ksampler
    samples = comfy.sample.sample(model, noise, steps, cfg, sampler_name, scheduler, positive, negative, latent_image,
  File "H:\ComfyUI_windows_portable\ComfyUI\comfy\sample.py", line 93, in sample
    samples = sampler.sample(noise, positive_copy, negative_copy, cfg=cfg, latent_image=latent_image, start_step=start_step, last_step=last_step, force_full_denoise=force_full_denoise, denoise_mask=noise_mask, sigmas=sigmas, callback=callback, disable_pbar=disable_pbar, seed=seed)
  File "H:\ComfyUI_windows_portable\ComfyUI\comfy\samplers.py", line 643, in sample
    positive = encode_adm(self.model, positive, noise.shape[0], noise.shape[3], noise.shape[2], self.device, "positive")
  File "H:\ComfyUI_windows_portable\ComfyUI\comfy\samplers.py", line 529, in encode_adm
    adm_out = model.encode_adm(device=device, **params)
  File "H:\ComfyUI_windows_portable\ComfyUI\comfy\model_base.py", line 183, in encode_adm
    clip_pooled = kwargs["pooled_output"]
KeyError: 'pooled_output'

Prompt executed in 5.21 seconds
```

The same workflow works with SD1.5-based models.  

Maybe someone can confirm my finding. 

Edit: Just realized that I used the Efficient's KSampler. I updated the flow to use the built-in one in stead. The result is the same though.

**Comment by WASasquatch** [comment]
Yeah that node is pre-widget converting so doesn't have pooled outouts for SDXL. Using a CLIPTextEncode encode is the same thing, and can resize it to same shape once text field is converted. I'll add pooled output for that node to the list of things to do.

**Comment by Rizwanali324** [comment]
i was  not able to see 
![image](https://github.com/user-attachments/assets/590eb3ca-25a3-45be-810b-b2ab5e1ddaa4)
